test/derivative/ClassicDeriv.h

```diff
@@ -16,23 +16,26 @@
 #define COM_CLASSICDERIV_H
 
 #ifndef NUM_RXNS
-#define NUM_RXNS 5000
+#define NUM_RXNS 500
 #endif
 #ifndef NUM_SPEC
-#define NUM_SPEC 2000
+#define NUM_SPEC 200
+#endif
+#ifndef NUM_CELLS
+#define NUM_CELLS 10
 #endif
 
 namespace jit_test {
 
 class ClassicDeriv {
 public:
   ClassicDeriv();
-  void Solve(const double *const state, double *const deriv);
+  void Solve(const double *const rateConst, const double *const state, double *const deriv);
   void WritePreprocessedFortran();
 
   int numRxns = NUM_RXNS;
   int numSpec = NUM_SPEC;
-  double rateConst[NUM_RXNS];
+  int numCell = NUM_CELLS;
```

Comment:
Similar to the previous comment, why do you remove `rateConst` as a member of the class?

Reply:
(see response to previous comment)